src/components/common/header/header.scss

```diff
@@ -163,7 +163,26 @@ header {
     }
 
     
-  }
+  } 
+
+  
+}  
+
+.popupdisplayHIDE{
+  display: none
+} 
+
+.popupdisplaySHOW{
+  display: flex; 
+  align-content: center;
+  align-self: center;
+  background-color: rgba(128, 128, 128, 0.39)
 }
```

Comment:
Have you looked at the popup on both mobile and desktop?

## Mobile
- The positioning is messed up
<img width="194" alt="Screen Shot 2020-10-11 at 10 13 49 AM" src="https://user-images.githubusercontent.com/14047327/95685086-965daf00-0baa-11eb-8be7-d4f0f4b26404.png">
- The "button" does not display
<img width="193" alt="Screen Shot 2020-10-11 at 10 15 02 AM" src="https://user-images.githubusercontent.com/14047327/95685093-a83f5200-0baa-11eb-89aa-1464b30ec015.png">

## Desktop
- Try taking the popup out of the document flow. It probably also should not be part of the ul. I'll make a PR to your branch to do some cleanup of leftover mess that I missed previously.
<img width="752" alt="Screen Shot 2020-10-11 at 10 16 08 AM" src="https://user-images.githubusercontent.com/14047327/95685160-fce2cd00-0baa-11eb-9001-1b5c331a527b.png">

Reply:
Take a look at this example: https://codepen.io/imprakash/pen/GgNMXO
About document flow: https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout/Normal_Flow

Reply:
Will refactor to use the https://codepen.io/imprakash/pen/GgNMXO pattern.